pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -90,10 +91,17 @@ func (srv PrometheusSrv) RouteGetRuleStatuses(c *models.ReqContext) response.Res
 
 		stateMap := srv.manager.GetStatesByRuleUID()
 		for _, rule := range alertRuleQuery.Result {
+			var queryStr string
+			encodedQuery, err := json.Marshal(rule.Data)
+			if err != nil {
+				queryStr = err.Error()
+			} else {
+				queryStr = string(encodedQuery)
+			}
 			alertingRule := apimodels.AlertingRule{
 				State:       "inactive",
 				Name:        rule.Title,
-				Query:       rule.DataToString(), // TODO: don't escape <>& etc
+				Query:       queryStr, // TODO: don't escape <>& etc
```

Comment:
@davidmparrott Do you understand this comment? Not sure what it means, why would we not escape certain things?

Reply:
The marshaller escapes certain characters by default, for the benefit of HTML IIRC. My thought was that it may be desirable to not do so when returning the query, to better enable someone to take the query string returned and programmatically do things like create similar conditions. It's also not essential